docs/experiment/apis/management-api/experiments.md

```diff
@@ -0,0 +1,1210 @@
+                    "key": "experiment-key",
+                    "name": "experiment-key",
+                    "description": "save button color",
+                    "enabled": false,
+                    "bucketingKey": "amplitude_id",
+                    "variants": [
+                        {
+                            "key": "control"
+                        },
+                        {
+                            "key": "treatment"
+                        }
+                    ],
+                    "rolloutWeights": {
+                        "control": 1,
+                        "treatment": 1
+                    },
+                    "targetSegments": [],
+                    "stickyBucketing": false,
+                    "state": "planning",
+                    "startDate": null,
+                    "endDate": null,
+                    "experimentType": "hypothesis-testing"
+                }
+            },
+            {
+                "createdAt": "2023-07-29T03:25:42.236Z",
+                "createdBy": <userId>,
+                "version": 1,
+                "flagConfig": {
+                    "id": <id>,
+                    "projectId": <projectId>,
+                    "deployments": [],
+                    "key": "experiment-key",
+                    "name": "experiment-key",
+                    "description": "",
+                    "enabled": false,
+                    "bucketingKey": "amplitude_id",
+                    "variants": [
+                        {
+                            "key": "control"
+                        },
+                        {
+                            "key": "treatment"
+                        }
+                    ],
+                    "rolloutWeights": {
+                        "control": 1,
+                        "treatment": 1
+                    },
+                    "targetSegments": [],
+                    "stickyBucketing": false,
+                    "state": "planning",
+                    "startDate": null,
+                    "endDate": null,
+                    "experimentType": "hypothesis-testing"
+                }
+            }
+        ]
+        ```
+
+------
+
+## Get version details
+
+```bash
+GET https://experiment.amplitude.com/api/1/experiments/{id}/versions/{versionId}
+```
+
+Fetch details of a specific version of an experiment.
+
+### Path variables
+
+|Name|Description|
+|---|----|
+|`id`| Required. String. Experiment's ID.|
+|`versionId`| Required. String. The version's ID.|
+
+### Response
+
+A successful request returns a `200 OK` response and a JSON object with details of the version.
+
+!!!example "Example cURL"
+    ```bash
+    curl --request GET \
+      --url 'https://experiment.amplitude.com/api/1/experiments/<id>/versions/<versionId>' \
+      --header 'Accept: application/json' \
+      --header 'Authorization: Bearer <management-api-key>'
+    ```
+
+???example "Experiment examples"
```

Comment:
For all examples responses on this page and the flags api page:
1. Title should be `Example response (click to open)`
2. Example can be removed from the tab layout (`===`)